src/views/CustomPages/FormFields/FormFieldsCustomPages.vue

```diff
@@ -0,0 +1,318 @@
+  import { onMounted, ref } from 'vue'
+
+  import FieldSwitchBlock from '@/templates/form-fields-inputs/fieldSwitchBlock'
+
+  const props = defineProps({
+    listEdgeConnectorsService: {
+      type: Function,
+      required: true
+    }
+  })
+
+  const STATUS_CODE_OPTIONS = [
+    {
+      code: '400',
+      name: '400: Bad Request'
+    },
+    {
+      code: '401',
+      name: '401: Unauthorized'
+    },
+    {
+      code: '403',
+      name: '403: Forbidden'
+    },
+    {
+      code: '404',
+      name: '404: Not Found'
+    },
+    {
+      code: '405',
+      name: '405: Method Not Allowed'
+    },
+    {
+      code: '406',
+      name: '406: Not Acceptable'
+    },
+    {
+      code: '408',
+      name: '408: Request Timeout'
+    },
+    {
+      code: '409',
+      name: '409: Conflict'
+    },
+    {
+      code: '410',
+      name: '410: Gone'
+    },
+    {
+      code: '411',
+      name: '411: Length Required'
+    },
+    {
+      code: '414',
+      name: '414: URI Too Long'
+    },
+    {
+      code: '415',
+      name: '415: Unsupported Media Type'
+    },
+    {
+      code: '416',
+      name: '416: Range Not Satisfiable'
+    },
+    {
+      code: '425',
+      name: '425: Too Early'
+    },
+    {
+      code: '426',
+      name: '426: Upgrade Required'
+    },
+    {
+      code: '429',
+      name: '429: Too Many Requests'
+    },
+    {
+      code: '431',
+      name: '431: Request Header Fields Too Large'
+    },
+    {
+      code: '500',
+      name: '500: Internal Server Error'
+    },
+    {
+      code: '501',
+      name: '501: Not Implemented'
+    },
+    {
+      code: '502',
+      name: '502: Bad Gateway'
+    },
+    {
+      code: '503',
+      name: '503: Service Unavailable'
+    },
+    {
+      code: '504',
+      name: '504: Gateway Timeout'
+    },
+    {
+      code: '505',
+      name: '505: HTTP Version Not Supported'
+    }
+  ]
+
+  const { value: name } = useField('name')
+  const { value: edgeConnectorId } = useField('edgeConnectorId')
+  const { value: isActive } = useField('isActive')
+  const { value: isDefault } = useField('isDefault')
+  const { push: pushPage, remove: removePage, fields: pages } = useFieldArray('pages')
+
+  const pageInitialState = {
+    code: 400,
+    ttl: 0,
+    uri: '',
+    customStatusCode: 0
+  }
+
+  const addNewPage = () => {
+    pushPage(pageInitialState)
+  }
+
+  const edgeConnectorsList = ref([])
+
+  const getEdgeConnectors = async () => {
+    edgeConnectorsList.value = await props.listEdgeConnectorsService()
+  }
+
+  onMounted(async () => {
+    // await getEdgeConnectors()
+    addNewPage()
+  })
+
+  const removePageFromList = async (index) => {
+    await removePage(index)
+  }
+
+  const isDefaultPage = (index) => index === 0
+</script>
+
+<template>
+  <FormHorizontal
+    title="Custom Page"
+    description="Customize error pages and cache TTL based on the HTTP status code received from the origin."
+    :isDrawer="true"
```

Comment:
```suggestion
    isDrawer
```
Verifica acho que passando somente a flag "isDrawer ele já tem o comportamento correto.